src/libData/AccountData/Account.cpp

```diff
@@ -27,7 +28,7 @@ Account::Account() {}
 
 Account::Account(const vector<unsigned char>& src, unsigned int offset)
 {
-    if (Deserialize(src, offset) != 0)
+    if (DeserializeAddOffset(src, offset) < 0)
```

Comment:
Does this mean there are multiple positive values for non-error return? Could just use `!=` to make it clear that the normal return code is just `0`